WebSharper.SweetAlert/WebSharper.SweetAlert.fsproj

```diff
@@ -1,13 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <WebSharperProject>Extension</WebSharperProject>
+    <WebSharperProject>binding</WebSharperProject>

```

Comment:
If we have the wsconfig.json, we can remove the WebSharper* tags from the project file